packages/client/hmi-client/src/components/widgets/VegaChart.vue

```diff
@@ -32,18 +44,47 @@ const props = withDefaults(
 		 */
 		intervalSelectionSignalNames?: string[];
 		config?: Config | null;
+		/**
+		 * Whether to show the expand button.
+		 * If a function is provided, it will be called before expanding the chart, and the returned spec will be used for the expanded chart.
+		 */
+		expand?: boolean | ((spec: VisualizationSpec) => VisualizationSpec);
 	}>(),
 	{
 		areEmbedActionsVisible: true,
 		intervalSelectionSignalNames: () => [],
-		config: null
+		config: null,
+		expand: false
```

Comment:
Nit pick: Is it worth to rename to `expandable` ?

Reply:
sure